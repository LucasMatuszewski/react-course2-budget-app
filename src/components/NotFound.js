import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
    <div>Error 404 - <Link to="/">go to help page</Link></div>
);

export default NotFound;