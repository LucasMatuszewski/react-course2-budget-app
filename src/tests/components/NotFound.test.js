import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render Not Found - Error 404', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});
