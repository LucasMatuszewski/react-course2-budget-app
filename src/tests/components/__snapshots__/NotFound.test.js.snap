// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found - Error 404 1`] = `
<div>
  Error 404 - 
  <Link
    replace={false}
    to="/"
  >
    go to help page
  </Link>
</div>
`;
